File: app/src/main/java/com/cryptojobboard/cryptojobboard/MainActivity.java
package com.cryptojobboard.cryptojobboard;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.support.v7.app.AppCompatActivity;

import com.cryptojobboard.cryptojobboard.feed.FeedFragment;
import com.cryptojobboard.cryptojobboard.rating.FiveStarsFragment;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        loadFragment(new FiveStarsFragment());
    }

    private void loadFragment(Fragment fragment) {
        // Create a fragment manager
        getSupportFragmentManager()
                .beginTransaction()
                .replace(R.id.frameLayout, fragment)
                .commit();
    }

}

File: app/src/main/java/com/cryptojobboard/cryptojobboard/MainActivity.kt
package com.cryptojobboard.cryptojobboard

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.support.v4.app.Fragment
import com.cryptojobboard.cryptojobboard.feed.FeedFragment

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        loadFragment(FeedFragment())
    }

    private fun loadFragment(fragment: Fragment) {
        // create a FragmentManager
        val fm = supportFragmentManager
        // create a FragmentTransaction to begin the transaction and replace the Fragment
        val fragmentTransaction = fm.beginTransaction()
        // replace the FrameLayout with new Fragment
        fragmentTransaction.replace(R.id.frameLayout, fragment)
        fragmentTransaction.commit() // save the changes
    }

}
